iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/LogicalPlanBuilder.java

```diff
@@ -192,27 +196,37 @@ public LogicalPlanBuilder planRawDataSource(
             .collect(Collectors.toList());
     List<PartialPath> groupedPaths = MetaUtils.groupAlignedSeries(selectedPaths);
     for (PartialPath path : groupedPaths) {
-      if (path instanceof MeasurementPath) { // non-aligned series
+      if (path instanceof MeasurementPath) {
+        // non-aligned series
+        // TODO: push down value filter
         SeriesScanNode seriesScanNode =
             new SeriesScanNode(
-                context.getQueryId().genPlanNodeId(), (MeasurementPath) path, scanOrder);
-        seriesScanNode.setTimeFilter(timeFilter);
-        // TODO: push down value filter
-        seriesScanNode.setValueFilter(timeFilter);
+                context.getQueryId().genPlanNodeId(),
+                (MeasurementPath) path,
+                scanOrder,
+                timeFilter,
+                timeFilter,
```

Comment:
twice timeFilter paremeter?

Reply:
yes, value filter can not be pushed down now, and we will implement it in few months.